js/windowMgr.js

```diff
@@ -224,10 +224,11 @@ function doCreateMainWindow(initialUrl, initialBounds) {
         initCrashReporterRenderer(mainWindow, { process: 'render | main window' });
 
         url = mainWindow.webContents.getURL();
-        if (isCustomTitleBarEnabled) {
+        if (isCustomTitleBarEnabled || isWindows10()) {
```

Comment:
shouldn't this be `isCustomTitleBarEnabled && isWindows10()`

Reply:
No, as we need show `native title bar with custom menu` for Windows 10 whether or not the `isCustomTitleBarEnabled` is enabled